app/Instructor/excel/postExcelAssignments.php

```diff
@@ -0,0 +1,177 @@
+		
+		$promptfileData = mysqli_real_escape_string($conn,file_get_contents($_FILES["submittedFile"]["tmp_name"]));
+		$prmptfileType = mysqli_real_escape_string($conn,$_FILES["submittedFile"]["type"]);
+		$promptfileSize = mysqli_real_escape_string($conn,$_FILES['submittedFile']['size']);
+		$keydata =array();
+		$keydata =readExcel($filePath);
+		$promptData =array();
+		$promptData=readExcel($subfilePath);
+		$rootPath = dirname(__FILE__);
+		$target_file = $rootPath . "/" . basename($_FILES["submittedFile"]["name"]);
+		move_uploaded_file($_FILES["submittedFile"]["tmp_name"], $target_file);
+		
+			$query ="insert into excelassignments(assignmentID,promptFile,keyFile,userVariableCell,acceptableExt,promptFileType,promptFileSize,promptFilePath) values ('$assignmentID','$promptfileData','$keyfileData','$keyCell', '.xlsx','" . mysqli_real_escape_string($conn,$_FILES["submittedFile"]["type"]) . "','$promptfileSize','" . mysqli_real_escape_string($conn,$target_file) . "')";
+
+			if($conn->query($query) === TRUE)
+	{
+	
+	$r1 =count($keydata);
+			$d= 0;$c = 0;$diffCount =0;
+			$diffData =array();	
+			echo '<div class="prev-update" >';
+			echo '<center>';
+			echo '<fieldset>';
+			echo '<h1 id="add-h1">Differnce found :</h1>';
+			echo '</br>';
+			echo '<table>' . "\n";
+			echo '<tr>' ;	
+			echo '<td>' . "S. No" . '</td>' .'<td>' . "Cell" . '</td>' .'<td>' . "Value" . '</td>' .'<td>' . "Formula" . '</td>'  . "\n";
+			for ( $i = 0; $i < $r1; $i++)
+			{	
+				$c1 =count($keydata[$i]);
+
+
+				$d =0;
+				for ( $j= 0; $j< $c1; $j++)
+				{	
+					if(isset($keydata[$i][$j]))
+					{
+						if($keydata[$i][$j] != '')
+						{
+							if( $keydata[$i][$j] == $promptData[$i][$j])
+							{
+									
+							}
+							else
+							{
+								echo '<tr>' ;	
+								$diffData[$c][$d] = [
+								'Value' => $keydata[$i][$j]['Value'],
+								'Index' => $keydata[$i][$j]['Index'],
+								'CalculateValue' => $keydata[$i][$j]['CalculateValue']
+								];
+								$query = "insert into differences(assignmentID,Cell,KeyFormula,Status,PotentialPoints,Type,Value) values('$assignmentID','{$diffData[$c][$d]['Index']}','{$diffData[$c][$d]['Value']}','A',0.2,'.xlsx','{$diffData[$c][$d]['CalculateValue']}')";
+								$conn->query($query);
+								echo '<td>' . ($diffCount+1) . '</td>' .'<td>' . $diffData[$c][$d]['Index'] . '</td>' .'<td>' . $diffData[$c][$d]['CalculateValue'] . '</td>'.'<td>' . $diffData[$c][$d]['Value'] . '</td>'  . "\n";			
+								$d++;$diffCount++;
+							}
+						}	
+					}
+					
+				}
+				echo '</tr>' ;
+				$c++;
+			}
+			?>
+			
+ </table>  <br/>
+ 			 </fieldset>
+			 </center>
+			 </div>
+			 <br/>
+			 </center>
+			 </div>
+</body>
+</html>
+<?php
+			$message = "Total of " . $diffCount . " differences saved in database" ;
+			echo "<script type='text/javascript'>alert('$message');</script>";
+				}
+	
+	else {
+	    echo "An error. Contact the adminstrator. Or go <a href='classHome.php'>home</a>.";
```

Comment:
I end up in one of these two else statements... i'm not sure where to start debugging

Reply:
Yes, currently it handles only formatted file only. I will handle this case in next update.